CHANGELOG.md

```diff
@@ -1,14 +1,79 @@
 # Changelog for Mapbox Maps SDK v10 for Android
 
 Mapbox welcomes participation and contributions from everyone.
-# 10.7.0
+
+# main
+## Features ✨ and improvements 🏁
+* Introduce a callback to be invoked when the device compass sensors need to be re-calibrated. ([1513](https://github.com/mapbox/mapbox-maps-android/pull/1513))
+
+## Bug fixes 🐞
+* Support altitude interpolation in location component, and pass through GPS altitude information from the DefaultLocationProvider. ([1478](https://github.com/mapbox/mapbox-maps-android/pull/1478))
```

Comment:
Are these `main` changes needed in v10.7 branch? Since the mentioned PRs are not present in this branch

Reply:
yeah, guess better remove them